src/foam/nanos/notification/NotificationMessageModal.js

```diff
@@ -5,70 +5,73 @@
  */
 
 foam.CLASS({
-    package: 'foam.nanos.notification',
-    name: 'NotificationMessageModal',
-    extends: 'foam.u2.View',
-  
-    documentation: 'View for displaying notification message',
+  package: 'foam.nanos.notification',
+  name: 'NotificationMessageModal',
+  extends: 'foam.u2.View',
 
-    css: `
-      ^ {
-        background-color: #fff;
-        width: 60vw;
-        max-height: 80vh;
-        overflow-y: auto;
-      }
-      ^ .main {
-        padding: 8px 24px 24px;
-      }
-      ^ .bold-label {
-        font-size: 16px;
-        font-weight: bold;
-        padding-bottom: 8px;
-      } 
-    `,
+  documentation: 'View for displaying notification message',
 
-    requires: [
-      'foam.u2.layout.Rows',
-      'foam.u2.layout.Grid',
-      'foam.u2.ModalHeader'
-    ],
+  css: `
+    ^ {
+      background-color: #fff;
+      width: 60vw;
+      max-height: 80vh;
+      overflow-y: auto;
+    }
+    ^ .main {
+      padding: 8px 24px 24px;
+    }
+    ^ .bold-label {
+      font-size: 16px;
+      font-weight: bold;
+      padding-bottom: 8px;
+    }
+    ^ .message {
+      white-space: pre-line;
+    } 
+  `,
 
-    messages: [
-      { name: 'NATIFICATION_MSG', message: 'Notification' },
-      { name: 'CREATED_MSG', message: 'Created' },
-      { name: 'MESSAGE_MSG', message: 'Message' }
-    ],
-  
-    properties: [
-      'data',
-      'created',
-      'description'
-    ],
-  
-    methods: [
-      function initE() {
-        this.SUPER();
+  requires: [
+    'foam.u2.layout.Rows',
+    'foam.u2.layout.Grid',
+    'foam.u2.ModalHeader'
+  ],
 
-        this.created = this.data.created.toUTCString();
-        this.description = this.data.body;
-        
-        this
-        .addClass(this.myClass())
-        .tag(this.ModalHeader.create({
-          title: this.NATIFICATION_MSG
-        }))
-        .start()
-          .addClass('main')
-            .start(this.Rows).style({padding: '24px 0px'})
-              .start().addClass('bold-label').add(this.CREATED_MSG).end()
-              .start().add(this.created).end()
-            .end()
-            .start(this.Rows).style({padding: '24px 0px'})
-              .start().addClass('bold-label').add(this.MESSAGE_MSG).end()
-              .start().add(this.description).end()
-            .end()
-        .end();
-      }
-    ]
-  });
+  messages: [
+    { name: 'NATIFICATION_MSG', message: 'Notification' },
+    { name: 'CREATED_MSG', message: 'Created' },
+    { name: 'MESSAGE_MSG', message: 'Message' }
+  ],
+
+  properties: [
+    'data',
+    'created',
+    'description'
+  ],
```

Comment:
Notifications also have a 'Title' property and a severity like WARN and ERROR and anything else is considered INFO for the user so a way to show that would be cool. The log level is an enum anyway so that should be easy. The title can just go before created.

Reply:
Also could be a topic for discussion if we don't want to make those changes now

Reply:
Yes, as this particular one would be 'WARN'. 
I believe the loglevel is only used when displaying toast notifications, and not presently in the notification citation view layout.  Perhaps @sarthak-marwaha you can review the citation view.  And considering that, we'll push this PR now and update after the review.

Reply:
That sounds good to me.